### ISSUE TITLE
OL6 causes rollup warnings

### ISSUE THREAD

**Issue by simon04** [openlayers/openlayers#10245]
**Describe the bug**
When bundling openlayers 6 with [rollup](https://rollupjs.org/), a few warnings are generated.

**To Reproduce**
```
$ git clone https://github.com/openlayers/ol-rollup
$ cd ol-rollup
$ yarn
$ yarn add ol
$ yarn run build
```

The following warnings are printed to the console:
```
$ rollup --config rollup.config.js

main.js → bundle.js...
(!) `this` has been rewritten to `undefined`
https://github.com/rollup/rollup/wiki/Troubleshooting#this-is-undefined
node_modules/ol/AssertionError.js
1: var __extends = (this && this.__extends) || (function () {
                    ^
2:     var extendStatics = function (d, b) {
3:         extendStatics = Object.setPrototypeOf ||
...and 1 other occurrence
node_modules/ol/Collection.js
1: var __extends = (this && this.__extends) || (function () {
                    ^
2:     var extendStatics = function (d, b) {
3:         extendStatics = Object.setPrototypeOf ||
...and 1 other occurrence
node_modules/ol/Feature.js
1: var __extends = (this && this.__extends) || (function () {
                    ^
2:     var extendStatics = function (d, b) {
3:         extendStatics = Object.setPrototypeOf ||
...and 1 other occurrence

...and 100 other files
```

**Expected behavior**
No ``this` has been rewritten to `undefined`` warning appears.

Relates to: https://github.com/openlayers/ol-rollup/pull/2

**Comment by ahocevar** [comment]
These warnings come from the transpiled code, which was done by bublé in previous version, but is done by the TypeScript compiler now.

According to https://stackoverflow.com/questions/43556940/rollup-js-and-this-keyword-is-equivalent-to-undefined, they can be ignored. In openlayers/ol-rollup, we could add a custom `onwarn` hook to get rid of this warning. @simon04, would you be able to update openlayers/ol-rollup#2 accordingly?

**Comment by DaniEll-AT** [comment]
I added a custom `onwarn` hook in openlayers/ol-rollup#2.
Suppressing these warnings should be ok (now that you confirmed my educated guess from openlayers/ol-rollup#1).

**Comment by ahocevar** [comment]
Fixed with openlayers/ol-rollup#2.
